examples/cpp/build.gradle

```diff
@@ -1,60 +1,80 @@
+import edu.wpi.first.gradlerio.deploy.roborio.RoboRIO
+
 plugins {
     id "cpp"
     id "google-test-test-suite"
-    id "edu.wpi.first.GradleRIO" version "2022.0.0-alpha-2"
+    id "edu.wpi.first.GradleRIO" version "2022.0.0"
 }
 
 // Define my targets (RoboRIO) and artifacts (deployable files)
-// This is added by GradleRIO's backing project EmbeddedTools.
+// This is added by GradleRIO's backing project DeployUtils.
 deploy {
     targets {
-        roboRIO("roborio") {
-            // Team number is loaded either from the .wpilib/wpilib_preferences.json
-            // or from command line. If not found an exception will be thrown.
-            // You can use getTeamOrDefault(team) instead of getTeamNumber if you
-            // want to store a team number in this file.
-            team = frc.getTeamNumber()
-        }
-    }
-    artifacts {
-        frcNativeArtifact('frcCpp') {
-            targets << "roborio"
-            component = 'frcUserProgram'
-            // Debug can be overridden by command line, for use with VSCode
-            debug = frc.getDebugOrDefault(false)
-        }
-        // Built in artifact to deploy arbitrary files to the roboRIO.
-        fileTreeArtifact('frcStaticFileDeploy') {
-            // The directory below is the local directory to deploy
-            files = fileTree(dir: 'src/main/deploy')
-            // Deploy to RoboRIO target, into /home/lvuser/deploy
-            targets << "roborio"
-            directory = '/home/lvuser/deploy'
+        roborio(RoboRIO) {
+            team = project.frc.getTeamNumber()
+            debug = true
+
+            frcNativeArtifact('frcCpp') {
+            }
         }
+        // frc {
+        //     roboRIO("roborio") {
+        //         // Team number is loaded either from the .wpilib/wpilib_preferences.json
+        //         // or from command line. If not found an exception will be thrown.
+        //         // You can use getTeamOrDefault(team) instead of getTeamNumber if you
+        //         // want to store a team number in this file.
+        //         team = project.frc.getTeamNumber()
+        //     }
+        // }
     }
+    // artifacts {
+    //     frc {
+    //         frcNativeArtifact('frcCpp') {
+    //             target = "roborio"
+    //             component = 'frcUserProgram'
+    //             // Debug can be overridden by command line, for use with VSCode
+    //             debug = project.frc.getDebugOrDefault(false)
+    //         }
+    //     }
+        // // Built in artifact to deploy arbitrary files to the roboRIO.
+        // fileTreeArtifact('frcStaticFileDeploy') {
+        //     // The directory below is the local directory to deploy
+        //     files = fileTree(dir: 'src/main/deploy')
+        //     // Deploy to RoboRIO target, into /home/lvuser/deploy
+        //     targets << "roborio"
+        //     directory = '/home/lvuser/deploy'
+        // }
+    //}
```

Comment:
Lots of commented lines here... what's the purpose?

Reply:
Just so I could have the original templates. REmoved them.